src/users/users.service.ts

```diff
@@ -63,7 +63,7 @@ export class UsersService {
       userName: user.userName,
       displayName: user.displayName,
       photo: this.getPhotoUrl(user),
-      email: user.email,
+      email: user.email ?? '',
```

Comment:
Should  we make `email` optional in `UserInfoDto` instead?

`null` is probably clearer than an empty string, when the user has no mail address.

Reply:
This was (as I already stated in the commit message) in an effort to be consistent with photoUrl…